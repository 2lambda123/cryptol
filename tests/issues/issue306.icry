:exhaust \(x:[6]) -> x <<< (0x10000000000000000 : [68]) == x <<< 4
:exhaust \(x:[6]) -> x >>> (0x10000000000000000 : [68]) == x >>> 4
:exhaust \(x:[8]) -> x << (0x10000000000000000 : [68]) == 0
:exhaust \(x:[8]) -> x >> (0x10000000000000000 : [68]) == 0
:exhaust \(x:[8]) -> x << (0x8000000000000000 : [64]) == 0
:exhaust \(x:[8]) -> x >> (0x8000000000000000 : [64]) == 0
