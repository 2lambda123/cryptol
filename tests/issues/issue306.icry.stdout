Loading module Cryptol
Using exhaustive testing.
testing...passed 64 tests.
Q.E.D.
Using exhaustive testing.
testing...passed 64 tests.
Q.E.D.
Using exhaustive testing.
testing...passed 256 tests.
Q.E.D.
Using exhaustive testing.
testing...passed 256 tests.
Q.E.D.
Using exhaustive testing.
testing...passed 256 tests.
Q.E.D.
Using exhaustive testing.
testing...passed 256 tests.
Q.E.D.
